device/pcie/pci_device.hpp

```diff
@@ -152,6 +154,11 @@ class PCIDevice {
      */
     tt::ARCH get_arch() const { return arch; }
 
+    /**
+     * @return whether the system is protected from this device by an IOMMU
+     */
+    bool is_behind_iommu() const { return behind_iommu; }
```

Comment:
same as Bojan's comment for naming, just for function -> `is_iommu_enabled`?

Reply:
renamed